tsdoc/src/nodes/DocInlineTag.ts

```diff
@@ -95,4 +95,20 @@ export class DocInlineTag extends DocNode {
       this._closingDelimiterParticle!
```

Comment:
I'd say `getChildNodes` shouldn't be overridable and child classes instead should override something that here returns `_tagContentParticle`, but for them may return something more elaborate.

Reply:
Good suggestion

---
In reply to: [221106843](https://github.com/Microsoft/tsdoc/pull/74#discussion_r221106843) [](ancestors = 221106843)